src/dotnet/Rider.Plugins.EfCore/EfCoreHelper.cs

```diff
@@ -49,5 +57,25 @@ public static IEnumerable<IProject> GetSupportedStartupProjects(ISolution soluti
         private static bool StartupProjectPackagesInstalled(IProject project) =>
             project.GetInstalledPackage(EfCoreRequiredPackages.EfCoreToolsNugetId) != default
             || project.GetInstalledPackage(EfCoreRequiredPackages.EfCoreDesignNugetId) != default;
+
+        private static void FillStartupProjectsList(LinkedList<IProject> startupProjects, IProject project)
```

Comment:
Could you make the method signature like that?

```csharp
IEnumerable<IProject> GetReferencingStartupProjects(LinkedList<IProject> startupProjects, IProject project)
```
?

Then, `startupProjects` will be like that:

```csharp
var startupProjects = projectsWithInstalledNugetPacks
    .SelectMany(GetStartupProjects)
    .Distinct();
```